integration-tests/api/factories/simple-gift-card-factory.ts

```diff
@@ -27,6 +27,7 @@ export const simpleGiftCardFactory = async (
     region_id: data.region_id,
     value: data.value,
     balance: data.balance,
+    tax_rate: data.tax_rate,
```

Comment:
todo: tax_rate must be added to the data type above as well. if not provided, should we try to query the region from the provided id as it is required and then take the tax rate from there? in that case the tax rate become optional